lib/synapse/service_watcher/README.md

```diff
@@ -0,0 +1,58 @@
+Please, do not submit pull requests with new watchers, as Synapse and Nerve
```

Comment:
Let's give folks somewhere to contribute these though. Don't want to discourage releasing watchers.